### DISCUSSION TITLE
Automatic Refresh of search for events when added a new event

### DISCUSSION THREAD

**Discussion by michaelof** [FossifyOrg/General-Discussion#462] [Ideas]
Hi,

I frequently have to add e.g. weekly events for e.g. next month. Some sort of recurring events, weekly, but very floating, means these appointments have to be discussed an aligned to all other partcipant's own calendars.

To add my events, as everything except the date and time is usually the same, 

1. I'm SEARCHING for something I know is (hopefully :)) unique. 
2. Getting a list of all events, sorted ascending from first event ever, so have to scroll down to latest event   (asked here a while ago and got AFAI remember the answer that it's not possible to change sort order, fine)
3. open the latest event, click on "duplicate"
4. adjust date and time
5. save

Having done that, I'm again in the list of previously searched events. 
New event, although a match for filter, is NOT shown, list is not refreshed.

Would be very convenient if the list would be or could be re-populated at that point.

As not, currently I have to repeat also steps 1 and 2, as I "must" see all existing events to be sure that all weeks are covered.

Not sure If I'm the only one with this IMHO usual calendar requirements for calendar-driven people like me :)

**Comment by Aga-C** [comment]
It's a bug. Please fill out a [bug report](https://github.com/FossifyOrg/Calendar/issues/new?assignees=&labels=bug%2Cneeds+triage&projects=&template=bug_report.yml) for this.
